modules/modules.nix

```
@@ -178,7 +178,7 @@ let
     (loadModule ./services/polybar.nix { })
     (loadModule ./services/pulseeffects.nix { condition = hostPlatform.isLinux; })
     (loadModule ./services/random-background.nix { })
-    (loadModule ./services/redshift-gammastep/redshift.nix { })
+    (loadModule ./services/redshift-gammastep/redshift.nix { condition = hostPlatform.isLinux; })
```

Unfortunately, for backwards compatibility reasons, the redshift one need to stay without a condition.
```suggestion
    (loadModule ./services/redshift-gammastep/redshift.nix { })
```